config/styleguide-visual/config.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
     },
   },
   launchOptions: {
-    args: ['--no-sandbox', '--disable-setuid-sandbox'],
+    args: ['--no-sandbox', '--disable-setuid-sandbox', '--font-render-hinting medium'],
```

Comment:
While I didn't see evidence of this flag helping, I have a slight preference to keep it incase the default changes again.
https://github.com/GoogleChrome/puppeteer/issues/2410

Reply:
Might want to try with the assignment symbol `'--font-render-hinting=medium'`